<?xml version="1.0" ?><root>
<report><version>2</version><sessionid>af47562c3105dc9656f8fb9</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2023-12-05 22:03:00</createtime><buildsignature>Local.Unknown.Unknown.1.103.250.1020-1.300.000.174.Release</buildsignature><categoryid>tunable_perf.py:49</categoryid><desyncid>af47562c3105dc9656f8fb9</desyncid><systemconfig/><screenshot/><desyncdata>[BetterExceptions] 'NoneType' object has no attribute 'append' (AttributeError: 'NoneType' object has no attribute 'append')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 130, in wrapped&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 126, in finally_wrap&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 157, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 175, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 356, in c_api_client_connect&#13;&#10;  File "D:\Projects\Sims 4 Modding\Mods\TMex Mods 2022\Tmex-BetterExceptions\PublicUnlock.py", line 13, in _unlock&#13;&#10;  File "D:\Projects\Sims 4 Modding\Mods\TMex Mods 2022\Tmex-BetterExceptions\tmex_BetterExceptions.py", line 89, in tm_update_check_injection&#13;&#10;  File "D:\Projects\Sims 4 Modding\Mods\TMex Mods 2022\Tmex-BetterExceptions\tmex_BE_Injector.py", line 54, in wrapper&#13;&#10;  File "D:\Projects\Sims 4 Modding\Mods\TMex Mods 2022\Tmex-BetterExceptions\tmex_BetterExceptions.py", line 131, in tm_be_show_welcome&#13;&#10;  File "D:\Projects\Sims 4 Modding\Mods\TMex Mods 2022\Tmex-BetterExceptions\tmex_BE_UI.py", line 183, in be_show_notification&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\tunable.py", line 793, in __init__&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\tunable.py", line 319, in __init__&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\tunable_base.py", line 464, in __init__&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\tunable_perf.py", line 49, in register_for_cleanup&#13;&#10;AttributeError: 'NoneType' object has no attribute 'append'rtim=0&#13;&#10;Client Session Time: 16.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0x96d01090&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 13,79,0,5592&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19043 GPU: AMD Custom GPU 0405 (RADV VANGOGH) aticfx64.dll 32767.65535.65535.65535, GUID: 00000000-0004-0000-0000-000000000000&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>af47562c3105dc9656f8fb9</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2023-12-05 22:03:00</createtime><buildsignature>Local.Unknown.Unknown.1.103.250.1020-1.300.000.174.Release</buildsignature><categoryid>tunable_perf.py:49</categoryid><desyncid>af47562c3105dc9656f8fb9</desyncid><systemconfig/><screenshot/><desyncdata>[BetterExceptions] 'NoneType' object has no attribute 'append' (AttributeError: 'NoneType' object has no attribute 'append')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Lib\threading.py", line 885, in _bootstrap&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Lib\threading.py", line 917, in _bootstrap_inner&#13;&#10;  File "D:\Projects\Sims 4 Modding\Mods\TMex Mods 2022\Tmex-BetterExceptions\tmex_BetterExceptions.py", line 65, in run&#13;&#10;  File "D:\Projects\Sims 4 Modding\Mods\TMex Mods 2022\Tmex-BetterExceptions\tmex_BetterExceptions.py", line 544, in ExceptionAlert&#13;&#10;  File "D:\Projects\Sims 4 Modding\Mods\TMex Mods 2022\Tmex-BetterExceptions\tmex_BetterExceptions.py", line 518, in show_alert&#13;&#10;  File "D:\Projects\Sims 4 Modding\Mods\TMex Mods 2022\Tmex-BetterExceptions\tmex_BEAnalysis.py", line 397, in ShowErrorDialog&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\tunable.py", line 793, in __init__&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\tunable.py", line 319, in __init__&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\tunable_base.py", line 464, in __init__&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\tunable_perf.py", line 49, in register_for_cleanup&#13;&#10;AttributeError: 'NoneType' object has no attribute 'append'rtim=0&#13;&#10;Client Session Time: 16.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0x96d01090&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 13,79,0,5592&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19043 GPU: AMD Custom GPU 0405 (RADV VANGOGH) aticfx64.dll 32767.65535.65535.65535, GUID: 00000000-0004-0000-0000-000000000000&#13; &#10;</desyncdata></report>
</root>